packages/viewport/src/index.ts

```diff
@@ -89,38 +90,41 @@ function createMediaQueryList( args?: { min?: number; max?: number } ): QueryIte
 	return false;
 }
 
-const mediaQueryLists: Record< string, QueryItem > = {
-	'<480px': createMediaQueryList( { max: 480 } ),
-	'<660px': createMediaQueryList( { max: 660 } ),
-	'<782px': createMediaQueryList( { max: 782 } ),
-	'<800px': createMediaQueryList( { max: 800 } ),
-	'<960px': createMediaQueryList( { max: 960 } ),
-	'<1040px': createMediaQueryList( { max: 1040 } ),
-	'<1280px': createMediaQueryList( { max: 1280 } ),
-	'<1400px': createMediaQueryList( { max: 1400 } ),
-	'>480px': createMediaQueryList( { min: 480 } ),
-	'>660px': createMediaQueryList( { min: 660 } ),
-	'>782px': createMediaQueryList( { min: 782 } ),
-	'>800px': createMediaQueryList( { min: 800 } ),
-	'>960px': createMediaQueryList( { min: 960 } ),
-	'>1040px': createMediaQueryList( { min: 1040 } ),
-	'>1280px': createMediaQueryList( { min: 1280 } ),
-	'>1400px': createMediaQueryList( { min: 1400 } ),
-	'480px-660px': createMediaQueryList( { min: 480, max: 660 } ),
-	'660px-960px': createMediaQueryList( { min: 660, max: 960 } ),
-	'480px-960px': createMediaQueryList( { min: 480, max: 960 } ),
+// @todo We should algin the behavior with the `break-*` mixins from Gutenberg to include minimums
+// See https://github.com/WordPress/gutenberg/blob/f1d0bd550f85f5fa4279a3fdb9a2b9c28a7544c6/packages/base-styles/_mixins.scss#L27
+const mediaQueryOptions: Record< string, QueryOption > = {
+	'<480px': { max: 480 },
+	'<660px': { max: 660 },
+	'<782px': { max: 782 },
+	'<800px': { max: 800 },
+	'<960px': { max: 960 },
+	'<1040px': { max: 1040 },
+	'<1280px': { max: 1280 },
+	'<1400px': { max: 1400 },
+	'>480px': { min: 480 },
+	'>660px': { min: 660 },
+	'>782px': { min: 782 },
+	'>800px': { min: 800 },
+	'>=960px': { min: 959 },
```

Comment:
Support the new breakpoint temporarily to align the behavior of the `break-*` mixins from Gutenberg.